### PR TITLE
Endpoint rent index

### PR DESCRIPTION
**Pull request by agustinbajo**
https://trello.com/c/uOSot0iR

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -18,7 +15,7 @@ gem 'sass-rails', '~> 5.0'
 gem 'uglifier', '>= 1.3.0'
 # See https://github.com/rails/execjs#readme for more supported runtimes
 # gem 'therubyracer', platforms: :ruby
-
+gem 'mini_racer', platforms: :ruby
```
Comment (leoncalermo) [inline Gemfile:18]: what's this gem for ?

File: Gemfile [hunk 2]
```diff
@@ -32,42 +29,44 @@ gem 'jbuilder', '~> 2.5'
...
   # Adds support for Capybara system testing and selenium driver
   gem 'capybara', '~> 2.13'
-  gem 'selenium-webdriver'
   gem 'database_cleaner', '~> 1.7.0'
-  gem 'faker', '~> 1.9.1', :git => 'https://github.com/stympy/faker.git', :branch => 'master'
-  #gem 'factory_bot'
-  gem 'rspec', '~> 3.8.0'
+  gem 'faker', '~> 1.9.1', git: 'https://github.com/stympy/faker.git', branch: 'master'
+  gem 'selenium-webdriver'
+  # gem 'factory_bot'
```
Comment (leoncalermo) [inline Gemfile]: remove this one if its commented

File: app/serializers/user_serializer.rb [hunk 3]
```diff
@@ -0,0 +1,3 @@
+class UserSerializer < ActiveModel::Serializer
+  attributes :id, :first_name, :first_name, :last_name, :email
```
Comment (leoncalermo) [inline app/serializers/user_serializer.rb]: first_name its repeated

File: bin/rails [hunk 4]
```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env ruby
 begin
-  load File.expand_path('../spring', __FILE__)
+  load File.expand_path('spring', __dir__)
```
Comment (leoncalermo) [inline bin/rails:3]: why all of this changes ? 🤔

File: bin/bundle [hunk 5]
```diff
@@ -1,3 +1,3 @@
 #!/usr/bin/env ruby
-ENV['BUNDLE_GEMFILE'] ||= File.expand_path('../../Gemfile', __FILE__)
+ENV['BUNDLE_GEMFILE'] ||= File.expand_path('../Gemfile', __dir__)
```
Comment (leoncalermo) [inline bin/bundle:2]: and this changes 🤔  ?

File: bin/yarn [hunk 6]
```diff
@@ -2,10 +2,10 @@
 VENDOR_PATH = File.expand_path('..', __dir__)
 Dir.chdir(VENDOR_PATH) do
   begin
-    exec "yarnpkg #{ARGV.join(" ")}"
+    exec "yarnpkg #{ARGV.join(' ')}"
   rescue Errno::ENOENT
-    $stderr.puts "Yarn executable was not detected in the system."
-    $stderr.puts "Download Yarn at https://yarnpkg.com/en/docs/install"
+    warn 'Yarn executable was not detected in the system.'
+    warn 'Download Yarn at https://yarnpkg.com/en/docs/install'
```
Comment (leoncalermo) [inline bin/yarn:8]: and this 🤔  ?

File: config/application.rb [hunk 7]
```diff
@@ -1,16 +1,15 @@
 require_relative 'boot'
-
 require 'rails/all'
 
 # Require the gems listed in Gemfile, including any gems
 # you've limited to :test, :development, or :production.
 Bundler.require(*Rails.groups)
+# config.assets.initialize_on_precompile = false
```
Comment (leoncalermo) [inline config/application.rb]: remove it if you are not going to use it

File: public/assets/.sprockets-manifest-33d6a223bc4d69f1df15bf0a3b3936d9.json [hunk 8]
```diff
@@ -0,0 +1 @@
+{"files":{"application-eaae6e935a232c711d293b3e22bd0f8a986b9be0b39f70a49e7b29977ad329f0.js":{"logical_path":"application.js","mtime":"2018-09-27T14:54:48-03:00","size":85961,"digest":"eaae6e935a232c711d293b3e22bd0f8a986b9be0b39f70a49e7b29977ad329f0","integrity":"sha256-6q5uk1ojLHEdKTs+Ir0Piphrm+Czn3Cknnspl3rTKfA="},"application-f0d704deea029cf000697e2c0181ec173a1b474645466ed843eb5ee7bb215794.css":{"logical_path":"application.css","mtime":"2018-09-27T14:54:48-03:00","size":676,"digest":"f0d704deea029cf000697e2c0181ec173a1b474645466ed843eb5ee7bb215794","integrity":"sha256-8NcE3uoCnPAAaX4sAYHsFzobR0ZFRm7YQ+te57shV5Q="}},"assets":{"application.js":"application-eaae6e935a232c711d293b3e22bd0f8a986b9be0b39f70a49e7b29977ad329f0.js","application.css":"application-f0d704deea029cf000697e2c0181ec173a1b474645466ed843eb5ee7bb215794.css"}}
```
Comment (leoncalermo) [inline public/assets/.sprockets-manifest-33d6a223bc4d69f1df15bf0a3b3936d9.json]: this file again 👀

File: spec/controllers/api/v1/books_controller_spec.rb [hunk 9]
```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
+require 'support/shared_contexts/authenticated_user.rb'
+describe Api::V1::BooksController, type: :controller do
```
Comment (leoncalermo) [inline spec/controllers/api/v1/books_controller_spec.rb:3]: Why are Books Controller tests in the Rent controller PR ? 🤔

**Review by leoncalermo — CHANGES_REQUESTED** [review]
Be careful when you run a `bundle exec rubocop -a` without specifying any folders, because it analyzes the entire project and this happens 😅

File: Gemfile (hunk 1 above)
Comment (agustinbajo) [reply Gemfile:18]: This Gem Ramiro tolme me to install it  to be able to implemente Heroku

File: Gemfile (hunk 2 above)
Comment (agustinbajo) [reply Gemfile]: Removed

File: app/serializers/user_serializer.rb (hunk 3 above)
Comment (agustinbajo) [reply app/serializers/user_serializer.rb]: Fixed

File: public/assets/.sprockets-manifest-33d6a223bc4d69f1df15bf0a3b3936d9.json (hunk 8 above)
Comment (agustinbajo) [reply public/assets/.sprockets-manifest-33d6a223bc4d69f1df15bf0a3b3936d9.json]: Fixed

File: bin/rails (hunk 4 above)
Comment (agustinbajo) [reply bin/rails:3]: *1

File: bin/bundle (hunk 5 above)
Comment (agustinbajo) [reply bin/bundle:2]: *1

File: bin/yarn (hunk 6 above)
Comment (agustinbajo) [reply bin/yarn:8]: *1  Really I do not know, those changes I did not make. Could it be rubocop?

File: config/application.rb (hunk 7 above)
Comment (agustinbajo) [reply config/application.rb]: Fixed

**Review by leoncalermo — APPROVED** [review]
dont run`bundle exec rubocop --auto-correct` withouth specifying folders hahaha,
use this  `bundle exec rubocop --auto-correct app spec`, with that should be enough

**Pull request merged**